common/src/main/java/com/microsoft/identity/common/internal/commands/InteractiveTokenCommand.java

```diff
@@ -39,21 +43,53 @@
 @EqualsAndHashCode(callSuper = true)
 public class InteractiveTokenCommand extends TokenCommand {
     private static final String TAG = InteractiveTokenCommand.class.getSimpleName();
+    //Normally all tasks have an affinity unless configured explicitly for multi-window support to not have one
+    private boolean mHasTaskAffinity = true;
+    private int mTaskId = 0;
```

Comment:
nit:

@Getter 
@Accessors(prefix = "m")

?

Reply:
oh boy... let's take this up with Brian and the rest of the team.  I prefer not to do this myself... however it's part of the android naming guidelines

Reply:
I think you misunderstand.  Putting those two annotations on this class will create the accessors understanding that the member prefix is "m", and name the methods without it.